### ISSUE TITLE
Design | CH 31 | Redesign Form Flow

### ISSUE THREAD

**Issue by steele-lm** [department-of-veterans-affairs/va.gov-team#68007]
## Issue Description
_CAIA feedback on this issue is in #64604._

### Staging or Production Link
- [Staging Link](https://staging.va.gov/careers-employment/vocational-rehabilitation/apply-vre-form-28-1900/start)

### Prototype Link
- [Mural User Flows](https://app.mural.co/t/departmentofveteransaffairs9999/m/departmentofveteransaffairs9999/1684161597392/5c1f8869cf5d03b792c8b82cf19b39dc26c486c2?sender=u0e8ac1d6d7681bb7e1b80558)

### Test Users
- vets.gov.user+24@gmail.com
- [staging test user repository](https://github.com/department-of-veterans-affairs/va.gov-team-sensitive/blob/master/Administrative/vagov-users/mvi-staging-users.csv)

### Details
There are many interconnected errors on the first three pages of this form. 
- Questions bounce between Veteran information and Applicant information, but there are no questions to validate if the person applying for the benefit is the Veteran or a person associated with the Veteran
- Disclaimer for updates on the form altering pre-filled information is not in a logical place to benefit the user
- Section labeled 'Mailing address asks for address as well as other contact information
- Additional information section is a "junk drawer" of unrelated questions that should be more logically placed
- URL does not follow DSVA recommendations to provide insight into the form properly
- Steps of process are not properly broken down grouping pages that are not the same 'category' of information

### Proposed solution or next steps
- Decide what information is needed about the Veteran no matter if the person applying is the Veteran or a person affiliated with the Veteran. 
- Update and add necessary flow for identifying if the person applying for the benefit is the Veteran of associated with the Veteran (step numbers associated with the following pages will need to be adjusted depending on how you proceed here.)

Suggested flow redesign: (pending decisions on Veteran vs Applicant information)
Step 1 of 4: Applicant information | Veteran information (prefill pattern) <see [reference pattern](https://staging.va.gov/supporting-forms-for-claims/release-information-to-va-form-21-4142/introduction?postLogin=true)
Step 1 of 4: Applicant information | Additional information (education level)
Step 2 of 4: Applicant contact information | Mailing address (include Y/N for moving in 30 days)
Step 2 of 4: Applicant contact information (if Yes) upcoming address
Step 2 of 4: Applicant contact information | phone and email

- Review spacing Mailing address and "We'll send any important..." should be spaced similarly to Step X of Y & "We'll save your application..."
- Update disclaimer should be above the H3
- H3 should be sentence case
The question 'Are you moving in the next 30 days? should be moved to this screen (follow advice on Mural)
- Phone number and email should be on the next screen

URLs should all be reviewed and be descriptive of the screen and what the Veteran will be inputting on the screen recommend updating pages to follow the H3 Heading information

[Mural Board for proposed flow changes](https://app.mural.co/t/departmentofveteransaffairs9999/m/departmentofveteransaffairs9999/1684161597392/5c1f8869cf5d03b792c8b82cf19b39dc26c486c2?sender=u0311acf667e6504dbdb14420)

---
## Tasks
- [ ] Capture existing flow (screenshots)
- [ ] Create a simplified flow of the existing flow
- [ ] Redesign problem flows in Ch 31 form
- [ ] Content source of truth document for CAIA
- [ ] Obtain CAIA approval

## Acceptance Criteria
- [ ] Sketch file of design
- [ ] Content source of truth document for CAIA
- [ ] Assign to Laura Steele who will create a follow-on engineering ticket
---

**Comment by steele-lm** [comment]
CAIA requested a "content source of truth" for the Chapter 31 form. They prefer a Word doc posted to Sharepoint, but a Google Doc of GitHub doc will also work.

**Comment by steele-lm** [comment]
@fiorella-io I added the request for the content source of truth from CAIA to this ticket!
